src/Features/Core/Portable/InlineParameterNameHints/IInlineParamNameHintsService.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.InlineParameterNameHints
+{
+    struct NameAndSpan
+    {
+        public string _name;
+        public TextSpan _span;
+    }
+
+    internal interface IInlineParamNameHintsService : ILanguageService
+    {
+        Task<IEnumerable<NameAndSpan>> GetInlineParameterNameHintsAsync(Document document, TextSpan textSpan, CancellationToken cancellationToken = default);
```

Comment:
do we actually need a span ever?  i though this was just placed at the start of an argument.  in which case, we jsut need position (an int) right?

Reply:
Correct, this should just be a position.